### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by jinyuxiaoqiang** [Plachtaa/VALL-E-X#122]
You are using Python version 3.10.11
Segmentation fault (core dumped)

Python 3.10.11 (main, Apr  5 2023, 14:15:30) [GCC 7.5.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
^[[Aprint(torch.cuda.is_available())
True
>>> print(torch.__version__)
2.1.0+cu118
>>>

No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 18.04.6 LTS
Release:        18.04
Codename:       bionic

Python 3.10.11
Linux 4.15.0-201-generic #212-Ubuntu SMP Mon Nov 28 11:29:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Comment by jinyuxiaoqiang** [comment]
Package                   Version
------------------------- ------------
aiofiles                  23.2.1
altair                    5.1.2
anyio                     3.7.1
asttokens                 2.4.0
attrs                     23.1.0
backcall                  0.2.0
certifi                   2023.7.22
charset-normalizer        3.3.0
click                     8.1.7
cn2an                     0.5.22
contourpy                 1.1.1
cycler                    0.12.1
decorator                 5.1.1
einops                    0.7.0
encodec                   0.1.1
exceptiongroup            1.1.3
executing                 2.0.0
fastapi                   0.103.2
ffmpy                     0.3.1
filelock                  3.12.4
fonttools                 4.43.1
fsspec                    2023.9.2
gradio                    3.48.0
gradio_client             0.6.1
h11                       0.14.0
httpcore                  0.18.0
httpx                     0.25.0
huggingface-hub           0.17.3
idna                      3.4
importlib-resources       6.1.0
inflect                   7.0.0
ipython                   8.16.1
jedi                      0.19.1
jieba                     0.42.1
Jinja2                    3.1.2
joblib                    1.3.2
jsonschema                4.19.1
jsonschema-specifications 2023.7.1
kiwisolver                1.4.5
langid                    1.1.6
MarkupSafe                2.1.3
matplotlib                3.8.0
matplotlib-inline         0.1.6
mpmath                    1.3.0
networkx                  3.1
nltk                      3.8.1
numpy                     1.26.1
nvidia-cublas-cu12        12.1.3.1
nvidia-cuda-cupti-cu12    12.1.105
nvidia-cuda-nvrtc-cu12    12.1.105
nvidia-cuda-runtime-cu12  12.1.105
nvidia-cudnn-cu12         8.9.2.26
nvidia-cufft-cu12         11.0.2.54
nvidia-curand-cu12        10.3.2.106
nvidia-cusolver-cu12      11.4.5.107
nvidia-cusparse-cu12      12.1.0.106
nvidia-nccl-cu12          2.18.1
nvidia-nvjitlink-cu12     12.2.140
nvidia-nvtx-cu12          12.1.105
orjson                    3.9.9
packaging                 23.2
pandas                    2.1.1
parso                     0.8.3
pexpect                   4.8.0
pickleshare               0.7.5
Pillow                    10.1.0
pip                       23.2.1
proces                    0.1.7
prompt-toolkit            3.0.39
ptyprocess                0.7.0
pure-eval                 0.2.2
pydantic                  1.10.13
pydub                     0.25.1
Pygments                  2.16.1
pyparsing                 3.1.1
python-dateutil           2.8.2
python-multipart          0.0.6
pytz                      2023.3.post1
PyYAML                    6.0.1
referencing               0.30.2
regex                     2023.10.3
requests                  2.31.0
rpds-py                   0.10.6
scipy                     1.11.3
semantic-version          2.10.0
setuptools                68.0.0
six                       1.16.0
sniffio                   1.3.0
stack-data                0.6.3
starlette                 0.27.0
SudachiPy                 0.6.7
sympy                     1.12
tokenizers                0.14.1
toolz                     0.12.0
torch                     2.1.0+cu118
torchaudio                2.1.0+cu118
torchvision               0.16.0+cu118
tqdm                      4.66.1
traitlets                 5.11.2
triton                    2.1.0
typing_extensions         4.8.0
tzdata                    2023.3
Unidecode                 1.3.7
urllib3                   2.0.7
uvicorn                   0.23.2
vocos                     0.1.0
wcwidth                   0.2.8
websockets                11.0.3
wheel                     0.41.0
whisper                   1.1.10
